app/models/MongoDateTimeFormats.scala

```diff
@@ -32,7 +32,18 @@ trait MongoDateTimeFormats {
       "$date" -> localDate.atStartOfDay(ZoneOffset.UTC).toInstant.toEpochMilli
     )
 
+  implicit val localDateTimeRead: Reads[LocalDateTime] =
+    (__ \ "$date").read[Long].map { millis =>
+      Instant.ofEpochMilli(millis).atZone(ZoneOffset.UTC).toLocalDateTime
+    }
+
+  implicit val localDateTimeWrite: Writes[LocalDateTime] = (localDateTime: LocalDateTime) =>
+    Json.obj(
+      "$date" -> localDateTime.toEpochSecond(ZoneOffset.UTC) * 1000
```

Comment:
was curious about this, found "Instant.now(Clock.systemUTC).toEpochMilli", reminds how straightforward joda was ;)